Atomic/info.py

```diff
@@ -66,20 +66,29 @@ def __init__(self):
         super(Info, self).__init__()
         self.beu = BackendUtils()
 
+    def version_tty(self):
+        if self.args.debug:
+            util.write_out(str(self.args))
+        util.write_out(self.version())
```

Comment:
A cleaner way to do this would be to have an internal `_versions()` function that takes the stream as an argument.  Then then unit tests could pass `StringIO()`, the  tty version would use `sys.stdout`.